be/tool/beck.c

```diff
@@ -1250,11 +1250,14 @@ static struct m0_stob_ad_domain *emap_dom_find(const struct action *act,
 {
 	struct m0_stob_ad_domain *adom = NULL;
 	int			  i;
+	uint8_t			  ht_idx;
```

Comment:
uint8_t for index will make it restrictive, we can even have e.g 1024 hash entries, so this can be changed to uint32_t or uint16_t

Reply:
Handled in commit id 21da02f